src/network/message.rs

```diff
@@ -74,9 +81,7 @@ impl Encodable for CommandString {
     ) -> Result<usize, encode::Error> {
         let mut rawbytes = [0u8; 12];
         let strbytes = self.0.as_bytes();
-        if strbytes.len() > 12 {
-            return Err(encode::Error::NetworkCommandTooLong(self.0.clone().into_owned()));
-        }
+        debug_assert!(strbytes.len() <= 12);
```

Comment:
Are we sure that switching from error type to a panic case is a good practice?

However, I understand that with "erlang fail fast" paradigm this can be nice since devs will see that they did something wong — and getting command string from some external data w/o proper filtering is a very bad practice.

Reply:
you meant "before we have bugs". The fact that if the bugs will happen that may come unseen. But this consideration shouldnt' be a stopper here

Reply:
Bugs in our library should cause panics, not error returns.